code/modules/projectiles/projectile.dm

```diff
@@ -241,8 +242,12 @@
 	src.speed = speed
 	// Randomize speed by a small factor to help bullet animations look okay
 	// Otherwise you get a   s   t   r   e   a   m of warping bullets in same positions
-	src.speed *= (1 + (rand()-0.5) * 0.30) // 15.0% variance either way
-	src.speed = clamp(src.speed, 0.1, 100) // Safety to avoid loop hazards
+	if (randomize_speed)
+		src.speed *= (1 + (rand()-0.5) * 0.30) // 15.0% variance either way
+		src.speed = clamp(src.speed, 0.1, 100) // Safety to avoid loop hazards
```

Comment:
```suggestion
```
Clamps it again 3 lines down outside the `if` block.

Reply:
Thanks for the correction, missed that.